src/typeahead/typeahead-window.ts

```diff
@@ -56,13 +56,13 @@ export class NgbTypeaheadWindow {
    */
   @Output('select') selectEvent = new EventEmitter();
 
-  getActive() { return this.results[this._activeIdx]; }
+  getActive() { return this.results[this.activeIdx]; }
 
-  markActive(_activeIdx: number) { this._activeIdx = _activeIdx; }
+  markActive(activeIdx: number) { this.activeIdx = activeIdx; }
```

Comment:
Wait a moment. This directive is not exported, so this method should be `@internal`, because it's called from the template, right? And same for `select()`, right?